### PR TITLE
Add JDBC sample and samples index page

### PR DESCRIPTION
**Pull request by rakillen**
Internal Jira WDT-411.
Hold for release, this links to model help tool.

### CODE REVIEW

File: site/samples/jdbc.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## JDBC Sample
+
+This sample WDT domain model section includes configuration for two JDBC data sources.
+ 
+The data source `datasource-1` has a sparse configuration meant to set a minimal number of values. It is not necessary to include additional attributes and sub-folders, unless you intend to override default values.
```
Comment (rosemarymarano) [inline site/samples/jdbc.md:5]: override default values -> override the default values

File: site/samples/jdbc.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The data source `datasource-2` includes additional folders and attributes to provide further customization, including targeting information, connection pool configuration, and such. The cluster `cluster-1` should be defined elsewhere within this model, or should already exist in a domain that is being updated.  
+
+It is recommended that credential fields, such as `PasswordEncrypted`, should not be stored as clear text in the model. Those values can be referenced in a separate variables file or Kubernetes secret, or the model can be encrypted using [Encrypt Model Tool](../encrypt.md).
```
Comment (rosemarymarano) [inline site/samples/jdbc.md:9]: can be encrypted using [Encrypt Model Tool](../encrypt.md). -> can be encrypted using the [Encrypt Model Tool](../encrypt.md).

File: site/samples/jdbc.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+```yaml
+resources:
+    JDBCSystemResource:
+        'datasource-1':
+            JdbcResource:
+                DatasourceType: GENERIC
+                JDBCDriverParams:
+                    URL: 'jdbc:oracle:thin:@//localhost:1521/myDB'
+                    DriverName: oracle.jdbc.OracleDriver
+        'datasource-2':
```
Comment (mriccell) [inline site/samples/jdbc.md:20]: Why 2 datasources one without any Connection pool properties and one with?  Would it not be better to have only one with the Connection Pool properties?  If you are going to have 2 then have maybe a GENERIC and A GridLink DS.

File: site/samples/jdbc.md [hunk 4]
```diff
@@ -1,13 +1,8 @@
...
  
-The data source `datasource-1` has a sparse configuration meant to set a minimal number of values. It is not necessary to include additional attributes and sub-folders, unless you intend to override the default values.
-
-The data source `datasource-2` includes additional folders and attributes to provide further customization, including targeting information, connection pool configuration, and such. The cluster `cluster-1` should be defined elsewhere within this model, or should already exist in a domain that is being updated.  
-
-It is recommended that credential fields, such as `PasswordEncrypted`, should not be stored as clear text in the model. Those values can be referenced in a separate variables file or Kubernetes secret, or the model can be encrypted using the [Encrypt Model Tool](../encrypt.md).
-
+The first sample has a JDBC data source with a sparse configuration meant to set a minimal number of values. It is not necessary to include additional attributes and sub-folders, unless you intend to override the default values.
 ```yaml
 resources:
```
Comment (mriccell) [inline site/samples/jdbc.md:7]: Thjis sample needs properties that are normally updated by customers
JDBCConnectionPoolParams:
                    StatementCacheSize: 10
                    InitialCapacity: 0
                    MinCapacity: 0
                    TestTableName: SQL ISVALID
                    StatementCacheType: LRU
                    MaxCapacity: 5
                    ConnectionReserveTimeoutSeconds: 10
                JDBCDataSourceParams:
                    GlobalTransactionsProtocol: OnePhaseCommit
                    JNDIName: my.datasources.ds2

Comment (ddsharpe) [reply site/samples/jdbc.md:7]: That does not make sense @mriccell 
There are two examples here, the first, the minimum needed, and the second has the properties, no?

Comment (mriccell) [reply site/samples/jdbc.md:7]: I disagree, we should ask Steve Felts what are the most popular values that customers update.  I believe it is Max and Min number of connections, test connection on reserve, as well as properties dealing with the connections.

Comment (mriccell) [reply site/samples/jdbc.md:7]: You can remove 
JDBCDataSourceParams:
GlobalTransactionsProtocol: OnePhaseCommit
JNDIName: my.datasources.ds2

Comment (rakillen) [reply site/samples/jdbc.md:6]: I reduced this to just a single model, with all the fields we've discussed.

**Pull request merged**